lib/tapioca/compilers/symbol_table/symbol_generator.rb

```diff
@@ -131,11 +131,7 @@ def compile_object(name, value)
         def type_name_of(value)
           klass = class_of(value)
 
-          type_name = public_module?(klass) && name_of(klass) || "T.untyped"
-          # Range needs to be processed separately to be put in the T::Range[] form
-          type_name = "T::Range[#{type_name_of(T.cast(value, T::Range[T.untyped]).first)}]" if klass == Range
```

Comment:
Is there some change to Sorbet that prefers `Range` over `T::Range`? It still says to use `T::Range` in the [docs](https://sorbet.org/docs/stdlib-generics#docsNav).

Reply:
I guess this fix comes from https://github.com/sorbet/sorbet/pull/1309 ?

Reply:
Ideally we would like to generate the untyped version of these `Enumerable` generic types since we can't consistently be able to guess the type of the type-members. (Here we were looking at the type of the first entry and assuming it was the same for the whole, which makes sense for `Range` but not for other `Enumerable` types)

So the plan was to always use the "Ruby" type names, but Sorbet didn't support it for `Range`. We've since changed that in https://github.com/sorbet/sorbet/pull/1309 and as you can see from the comments on that PR, it is perfectly fine to use the naked types in files that are `typed: true`.

This change removes complexity from the generator so I prefer it. However, it does not strictly belong in this PR. But I'll let it slide for now. :)